README.md

```diff
@@ -5,4 +5,5 @@ Collection of PDF manipulation utilities
 # Developer setup
 
 - `./git-init-submodules.sh` to preload all the relevent submodules after a git clone
+- copy core classes from submodule javacommons-core (in JavaCommons-core/src/main/java/picoded/core)
```

Comment:
not so sure if copy is the right word : maybe "load the submodule for javacommons-core"

Reply:
updated the relevant note in [https://github.com/picoded/JavaCommons-pdf/pull/1/commits/fd634209055d33c199473d2d6f11fe964f8e5785](url)